package.json

```diff
@@ -12,7 +12,8 @@
     "url": "https://github.com/cypress-io/cypress-realworld-app/issues"
   },
   "engines": {
-    "node": ">=12.22.0"
+    "node": "^14.0.0 || ^16.0.0 || >=18.0.0",
+    "yarn": ">=1.19.0 < 2.0.0"
```

Comment:
This does not prevent installation with Yarn 3.5.1.
Did you try it out?

Reply:
See https://github.com/devoto13/yarn-plugin-engines#motivation

Yarn berry does not check `engines` unless the plugin https://github.com/devoto13/yarn-plugin-engines is installed, so the yarn check does not work well.

The plugin can however only be installed on Yarn berry, so there is no solution for this which covers all versions of Yarn and which can be set up in this repository.

I suggest to remove this line `"yarn"` since it is ineffective and misleading.

Reply:
@MikeMcC399  thank you for the review, I have addressed your feedback. Could you please review the changes again? Additionally, I'm considering adding the .npmrc file with engine-strict=true to enable strict enforcement of the engines field in the package.json file. If you don't mind, can I include that as part of this pull request?